presto-docs/src/main/sphinx/release/release-0.280.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.280
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where if a query was submitted before the resource group configuration had been loaded, all queries to the cluster would fail with a ``NullPointerException``. This fixes a race condition where a query can start constructing resource groups before the ``ResourceGroupConfigurationManager`` is loaded. 
+* Enhance ``GatherAndMergeWindows`` optimization rule by adding Frame bounds ([rank, rank])
```

Comment:
@feilong-liu can you please provide proper release note for this fix?

Reply:
Fix a bug in the ``GatherAndMergeWindows`` optimization which produces invalid plans when the output of a window function is used in the frame definition of another window function.